client/src/modules/distribution_center/distribution_key/distribution_key.js

```diff
@@ -46,22 +46,14 @@ function DistributionKeyController(DistributionCenters, Notify, uiGridConstants,
       headerCellFilter : 'translate',
     }, {
       field : 'rate',
-      type : 'number',
       displayName : 'TABLE.COLUMNS.RATE',
       headerCellFilter : 'translate',
-      cellClass : 'text-right',
-      footerCellClass : 'text-right',
-      cellFilter : 'number: 2',
-      footerCellFilter : 'number:2',
-      enableFiltering : true,
-      treeAggregationType : uiGridGroupingConstants.aggregation.SUM,
```

Comment:
This change is cool, please only restore **treeAggregationType : uiGridGroupingConstants.aggregation.SUM,**